lib/graphql.rb

```diff
@@ -61,6 +61,14 @@ def -@
       end
     end
   end
+
+  module StringMatchBackport
+    refine String do
+      def match?(pattern)
+        self =~ pattern
+      end
+    end
+  end
```

Comment:
This is just to let me use `String#match?` where  supported and avoid the string allocations of =~.